images-list

```diff
@@ -15,7 +15,7 @@ quay.io/coreos/prometheus-operator rancher/coreos-prometheus-operator v0.38.1
 quay.io/coreos/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.38.1
 quay.io/coreos/kube-state-metrics rancher/coreos-kube-state-metrics v1.9.5
 plugins/docker rancher/plugins-docker 18.09
-minio/minio rancher/minio-minio RELEASE.2020-02-07T23-28-16Z
```

Comment:
remove the extra `:RELEASE.2020-06-03T22-13-49Z` after `minio/minio`.

The syntax of the file is 
image source name, image destination name, desired tag